lib/rules/web/admin_console/4.1/projects.xyaml

```diff
@@ -30,7 +30,14 @@ set_project_description:
 send_delete_string:
   elements:
   - selector:
-      xpath: //input[@class='form-control']
+      xpath: //input[contains(@class,'form-control')]
     type: input
     op: send_keys <resource_name>
+click_button_if_needed:
+  action:
+    if_element:
+      selector:
+        xpath: //button[contains(text(),'<button_text>')]
```

Comment:
As you said `The console removed the Resource & Dashboard button on v4.2` but we still use the `Dashboard` steps in feature files, it doesn't look straightforward, why not just visiting the project URL?

Reply:
and the step `click_button_if_needed` looks not very meaningful

Reply:
`Resource` & `Dashboard` are different pages but with same url. So you have to click the `Dashboard` button to bring you to the project detail page on v4.1 console.

Reply:
I write like this `click_button_if_needed`because I want the feature file can compatible with all versions.